.vale/styles/RedHat/CaseSensitiveTerms.yml

```
@@ -12,6 +12,7 @@ swap:
   asciidoc|Asciidoc: AsciiDoc
   asciidoctor|AsciiDoctor: Asciidoctor
   '[nN]odejs|[nN]ode\.JS|node\.js': Node.js
+  basic authentication|Basic Authentication|basic auth|Basic auth|Basic Auth: Basic HTTP authentication (first instance) or Basic authentication
```

like `lower left|lower-left` in the TermsErrors rule:

```suggestion
  basic authentication|Basic Authentication|basic auth|Basic auth|Basic Auth: Basic HTTP authentication (first instance)|Basic authentication
```

---

Updated. Thanks.